src/lib.rs

```diff
@@ -167,3 +167,55 @@ mod parsing {
         }
     }
 }
+
+#[cfg(feature = "parsing")]
+pub mod parsing {
+    //! This module contains a set of exported nom parsers which can be used to
+    //! build nom parsers for custom grammars when used alongside the `syn_nom`
+    //! crate.
+    //!
+    //! `syn` uses its own custom fork of `nom`, `syn_nom`, which is smaller, and
+    //! thus improves build speeds. This should be used instead of `nom` when
+    //! building parsers using these parsers.
+
+    pub use macro_input::parsing::macro_input;
+
+    #[cfg(feature = "full")]
+    pub use krate::parsing::krate;
```

Comment:
I don't think this is one that people will need in their procedural macros, let's leave it out.